src/components/Header.js

```diff
@@ -1,12 +1,102 @@
+import {useEffect, useState} from 'react';
```

Comment:
`useEffect` is defined but never used -  please remove it ✌️

Reply:
Done.